src/styles/types.js

```diff
@@ -317,27 +318,27 @@ export type ThemeT = {|
 |};
 
 export type IconT = {
-  Alert?: React.Node,
-  ArrowDown?: React.Node,
-  ArrowLeft?: React.Node,
-  ArrowRight?: React.Node,
-  ArrowUp?: React.Node,
-  CheckIndeterminate?: React.Node,
-  Check?: React.Node,
-  ChevronLeft?: React.Node,
-  ChevronRight?: React.Node,
-  DeleteAlt?: React.Node,
-  Delete?: React.Node,
-  Filter?: React.Node,
-  Grab?: React.Node,
-  Menu?: React.Node,
-  Overflow?: React.Node,
-  Plus?: React.Node,
-  Search?: React.Node,
-  Spinner?: React.Node,
-  TriangleDown?: React.Node,
-  TriangleLeft?: React.Node,
-  TriangleRight?: React.Node,
-  TriangleUp?: React.Node,
-  Upload?: React.Node,
+  Alert?: typeof React.Component,
```

Comment:
Are there concrete implementations for each of these?

If we can, we should avoid `typeof React.Component' here and instead use the concrete types for each of these if they exist.  This is particularly useful if we want to constrain the prop types.